src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -75,6 +77,7 @@ void *torque_driver_alloc() {
     torque_driver_set_option(torque_driver, TORQUE_QDEL_CMD,
                              TORQUE_DEFAULT_QDEL_CMD);
     torque_driver_set_option(torque_driver, TORQUE_NUM_CPUS_PER_NODE, "1");
+    torque_driver_set_option(torque_driver, TORQUE_MEMORY_PER_JOB, "1g");
```

Comment:
Does 1g means default is 1GB?

Reply:
hmm, yes, but it is probably a bug, it should be "1gb"

Reply:
It is debated though if we should not provide any default for this.